addon/components/sl-grid.js

```diff
@@ -0,0 +1,660 @@
+import Ember from 'ember';
+import layout from '../templates/components/sl-grid';
+
+/**
+ * Valid values for the column definitions' `align` property
+ *
+ * @memberof module:components/sl-grid
+ * @enum {String}
+ */
+const COLUMN_ALIGN = {
+    LEFT: 'left',
+    RIGHT: 'right'
+};
+export { COLUMN_ALIGN };
+
+/**
+ * Valid values for the column definitions' `size` property
+ *
+ * @memberof module:components/sl-grid
+ * @enum {String}
+ */
+const COLUMN_SIZE = {
+    LARGE: 'large',
+    MEDIUM: 'medium',
+    SMALL: 'small'
+};
+export { COLUMN_SIZE };
+
+/**
+ * @module
+ * @augments ember/Component
+ */
+export default Ember.Component.extend({
+
+    // -------------------------------------------------------------------------
+    // Dependencies
+
+    // -------------------------------------------------------------------------
+    // Attributes
+
+    /** @type {String[]} */
+    classNameBindings: [ 'detailPaneOpen:details-open', 'loading:sl-loading' ],
```

Comment:
In some of the files in this changeset this list in horizontal, in others it is displayed vertically.